test/ig/IGExporter.IG.test.ts

```diff
@@ -0,0 +1,548 @@
+              },
+              name: 'Another Patient Example',
+              description: 'Another example of a Patient',
+              exampleBoolean: true // Usage set to Example sets this to true
+            }
+          ],
+          page: {
+            nameUrl: 'toc.html',
+            title: 'Table of Contents',
+            generation: 'html',
+            page: [
+              {
+                nameUrl: 'index.html',
+                title: 'Home',
+                generation: 'markdown'
+              }
+            ]
+          },
+          parameter: [
+            {
+              code: 'copyrightyear',
+              value: `${new Date().getFullYear()}+`
+            },
+            {
+              code: 'releaselabel',
+              value: 'CI Build'
+            },
+            {
+              code: 'path-history',
+              value: 'http://hl7.org/fhir/sushi-test/history.html'
+            }
+          ]
+        }
+      });
+    });
+  });
+
+  describe('#customized-ig', () => {
+    let pkg: Package;
+    let exporter: IGExporter;
+    let tempOut: string;
+    let fixtures: string;
+    let packageJSON: PackageJSON;
+    let config: Configuration;
+    let defs: FHIRDefinitions;
+
+    beforeAll(() => {
+      fixtures = path.join(__dirname, 'fixtures', 'customized-ig');
+      packageJSON = fs.readJSONSync(path.join(fixtures, 'package.json'));
+      tempOut = temp.mkdirSync('sushi-test');
+      defs = new FHIRDefinitions();
+      loadFromPath(
+        path.join(__dirname, '..', 'testhelpers', 'testdefs', 'package'),
+        'testPackage',
+        defs
+      );
+    });
+
+    beforeEach(() => {
+      config = {
+        filePath: path.join(fixtures, 'config.yml'),
+        id: 'sushi-test',
+        url: 'http://hl7.org/fhir/sushi-test',
+        version: '0.1.0',
+        name: 'sushi-test',
+        title: 'FSH Test IG',
+        description: 'Provides a simple example of how FSH can be used to create an IG',
+        dependencies: [
+          { packageId: 'hl7.fhir.r4.core', version: '4.0.1' },
+          { packageId: 'hl7.fhir.us.core', version: '3.1.0' },
+          { packageId: 'hl7.fhir.uv.vhdir', version: 'current' }
+        ],
+        status: null,
+        template: null,
+        fhirVersion: ['4.0.1'],
+        language: 'en',
+        publisher: 'James Tuna',
+        contact: [
+          {
+            name: 'Bill Cod',
+            telecom: [
+              { system: 'url', value: 'https://capecodfishermen.org/' },
+              { system: 'email', value: 'cod@reef.gov' }
+            ]
+          }
+        ],
+        license: 'CC0-1.0'
+      };
+      pkg = new Package(packageJSON, config);
+      exporter = new IGExporter(pkg, defs, path.resolve(fixtures, 'ig-data'), false);
+    });
+
+    afterAll(() => {
+      temp.cleanupSync();
+    });
+
+    it('should provide defaults for copyrightyear and releaselabel parameters', () => {
```

Comment:
I think maybe it shouldn't anymore.  I'll generate out some defaults to config.yaml in the case that the user has _no_ config.yaml in their project.  But if a config.yaml exists, we should just expect the user to set copyright year and releaselabel since any guesses we take will be wrong in some cases.